.vscode/cspell.json

```diff
@@ -234,6 +234,13 @@
         "Undelete"
       ]
     },
+    {
+      "filename": "**/sdk/synapse/**/*.md",
+      "words": [
+        "dotnetcli",
```

Comment:
Curious how this is used. Links (even markdown links) shouldn't matter for spelling, and if used in text, this doesn't seem an appropriate marketing term. If it is used in a general, conforming manner, maybe it should be global to the repo?

Reply:
You're right, it's repo-wide.  I'll make it global - thanks!

(For e.g. here: https://raw.githubusercontent.com/Azure/azure-sdk-for-net/main/sdk/storage/Azure.Storage.Common/README.md

![image](https://user-images.githubusercontent.com/21269347/144686931-d1da2c02-aaaf-4d87-8319-9b2e154a7253.png))

Reply:
Ah. I completely forgot about that. Someone did it a few months ago because it renders better on docs.microsoft.com.